### PR TITLE
Fix stats report

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: src/inverted_index.c [hunk 1]
```diff
@@ -1195,6 +1195,9 @@ int IndexBlock_Repair(IndexBlock *blk, DocTable *dt, IndexFlags flags, IndexRepa
     return -1;
   }
 
+  params->bytesBeforFix = blk->buf.offset;
+  ;
```
Comment (ashtul) [inline src/inverted_index.c]: remove

**Review by rafie — APPROVED** [review]

**Review by ashtul — COMMENTED** [review]
Amazing job!!
But why not remove `params->bytesCollected` or alternatively update it with `params->bytesBeforFix - params->bytesAfterFix`?

File: src/inverted_index.c (hunk 1 above)
Comment (MeirShpilraien) [reply src/inverted_index.c]: Already done

**Comment by MeirShpilraien** [comment]
> Amazing job!!
> But why not remove `params->bytesCollected` or alternatively update it with `params->bytesBeforFix - params->bytesAfterFix`?

I thought it might be useful for some other stuff in the future, this indicates the number of bytes removed from deleted documents. Don't see why it wrong to keep it.

**Review by ashtul — APPROVED** [review]

**Pull request merged**